src/pybind/mgr/dashboard/controllers/health.py

```diff
@@ -110,6 +110,55 @@
     'scrub_status': (str, '')
 })
 
+HEALTH_SNAPSHOT_SCHEMA = ({
+    'fsid': (str, 'Cluster filesystem ID'),
+    'health': ({
+        'status': (str, 'Overall health status'),
+        'checks': ({
+            '<check_name>': ({
+                'severity': (str, 'Health severity level'),
+                'summary': ({
+                    'message': (str, 'Human-readable summary'),
+                    'count': (int, 'Occurrence count')
+                }, 'Summary details'),
+                'muted': (bool, 'Whether the check is muted')
+            }, 'Individual health check object')
+        }, 'Health checks keyed by name'),
+        'mutes': ([str], 'List of muted check names')
+    }, 'Cluster health overview'),
+    'monmap': ({
+        'num_mons': (int, 'Number of monitors')
+    }, 'Monitor map details'),
+    'osdmap': ({
+        'in': (int, 'Number of OSDs in'),
+        'up': (int, 'Number of OSDs up'),
+        'num_osds': (int, 'Total OSD count')
```

Comment:
we are going to miss the down and out status? also the nearful and full osd counts?